### PR TITLE
fix links

### PR DESCRIPTION
**Pull request by SergeyParamoshkin**
**Description:** <Describe what has changed.>
Fixing a part links 
**Link to tracking Issue:** <Issue number if applicable>
#10973

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Comment by dmitryax** [comment]
@SergeyParamoshkin please sign the CLA and update PR header and description

File: Makefile.Common [hunk 1]
```diff
@@ -99,7 +99,7 @@ checklicense:
 .PHONY: checklinks
 checklinks:
 	command -v $(MDLINKCHECK) >/dev/null 2>&1 || { echo >&2 "$(MDLINKCHECK) not installed. Run 'npm install -g markdown-link-check'"; exit 1; }
-	find . -name \*.md -print0 | xargs -0 -n1 \
+	find . -name \*.md -print0 | sort | uniq | xargs -0 -n1 \
```
Comment (TylerHelmuth) [inline Makefile.Common:102]: Is this change related or is it an improvement to the checklinks target?

Comment (SergeyParamoshkin) [reply Makefile.Common:102]: it's small improvement for getting list in the same order

File: receiver/redisreceiver/testdata/config.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+# TODO: create issue
```
Comment (dmitryax) [inline receiver/redisreceiver/testdata/config.yaml:1]: This is already resolved https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11075, please rebase

File: extension/httpforwarder/testdata/config.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+http_forwarder:
+  ingress:
+    endpoint: localhost:7070
+  egress:
+    endpoint: http://target/
+    headers:
+      otel_http_forwarder: dev
+    timeout: 5s
```
Comment (dmitryax) [inline extension/httpforwarder/testdata/config.yaml:8]: Can you please submit another PR to add this file along with a test case similar to https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/11075

**Comment by dmitryax** [comment]
Also please link issues that are fixed in this PR. https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/10973 is only for cmd/configschema and already resolved
